utils/needs-manual-prs.js

```diff
@@ -0,0 +1,41 @@
+const fetch = require('node-fetch')
+const { ORGANIZATION_NAME, REPO_NAME } = require('../constants')
+
+// Fetch all PRs targeting 'master' that have a 'needs-manual/<branch>' label on them
+async function fetchNeedsManualPRs(branch) {
+  const baseUrl = `https://api.github.com/search/issues?`
+  
+  // cCnstruct queryString components
+  const repo = `repo:${ORGANIZATION_NAME}/${REPO_NAME}`
+  const type = `type:pr`
+  const state = `state:closed`
+  const label = `label:"needs-manual-bp/${branch}"`
+  
+  // Assemble final endpoint
+  const url = baseUrl + `q=${type}+${repo}+${state}+${label}`
```

Comment:
Could use Node.js built-in `querystring` module to serialize the params into your string.

See: https://nodejs.org/api/querystring.html

Reply:
Unfortunately in this case it wouldn't simplify too much because there's only one actual querystring, and the rest is GitHub's search endpoint logic not accounted for by that module,